cluster/gce/gci/configure-helper.sh

```diff
@@ -1175,12 +1175,6 @@ rules:
       - /version
       - /swagger*
 
-  # Don't log events requests.
```

Comment:
This needs to stay.  Maybe update the comment to include the reason to prevent future contributors from tripping up? eg:
```Don't log event requests because of performance impact.```

Reply:
Updated as requested.
Left everything else intact.